src/app/pages/quiz/quiz.component.ts

```diff
@@ -1,10 +1,19 @@
-import { Component } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 import { UIKitModule } from '../../ui-kit/ui-kit.module';
+import { ActivatedRoute } from '@angular/router';
 
 @Component({
   standalone: true,
   selector: 'app-quiz',
   templateUrl: './quiz.component.html',
   imports: [UIKitModule],
 })
-export class QuizComponent {}
+export class QuizComponent implements OnInit {
+  @Input() quiz: string = '';
+  constructor(private route: ActivatedRoute) { }
+  
+  ngOnInit(): void {
+    // eslint-disable-next-line dot-notation
+    this.quiz = this.route.snapshot.params['name'];
```

Comment:
can we use here this.route.snapshot.params.name to prevent disabling the rule?

Reply:
yes, we can, but if I use `this.route.snapshot.params.name`  I get this error: "Property 'name' comes from an index signature, so it must be accessed with ['name'].ts(4111)"